handler/src/test/java/io/netty/handler/ssl/ParameterizedSslHandlerTest.java

```diff
@@ -95,16 +94,11 @@
         return params;
     }
 
-    private final SslProvider clientProvider;
-    private final SslProvider serverProvider;
-
-    public ParameterizedSslHandlerTest(SslProvider clientProvider, SslProvider serverProvider) {
-        this.clientProvider = clientProvider;
-        this.serverProvider = serverProvider;
-    }
-
-    @Test(timeout = 480000)
-    public void testCompositeBufSizeEstimationGuaranteesSynchronousWrite()
+    @ParameterizedTest(name = "{index}: clientProvider={0}, {index}: serverProvider={1}")
```

Comment:
I think it's possible for annotations to refer to strings in static final fields, so we don't have to copy this one over and over.